File: seed/java-sdk/exhaustive/flat-package-layout/src/main/java/com/snippets/Example32.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.types.TypesObjectWithOptionalField;

public class Example32 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.endpointsObject()
                .endpointsObjectGetAndReturnWithOptionalField(
                        TypesObjectWithOptionalField.builder().build());
    }
}

File: seed/csharp-sdk/exhaustive/no-root-namespace-for-core-classes/Snippets/Example32.cs
using SeedApi;
using SeedApi.Core;

public partial class Examples
{
    public async Task Example32() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Enum.GetAndReturnEnumAsync(
            TypesWeatherReport.Sunny
        );
    }

}
